variables.tf

```
@@ -223,6 +223,18 @@ variable "snap_start" {
   default     = false
 }
 
+variable "replace_security_groups_on_destroy" {
+  description = "(Optional) When true, all security groups defined in vpc_security_group_ids will be replaced with the default security group after the function is destroyed. Set the replacement_security_group_ids variable to use a custom list of security groups for replacement instead."
+  type        = bool
+  default     = false
```

```suggestion
  default     = null
```

Let Terraform AWS provider decide which default value is best.

PS: I don't think there is an issue (e.g., BC break) if we use `true` as default. It doesn't seem to me that it will have bad consequences for users.

---

good shout, indeed it's best to leave the decision of a default to the provider :+1:

---

I've pushed the change